internal/scm/bitbucketcloud/bitbucket_cloud.go

```diff
@@ -52,14 +52,23 @@ func (bbc *BitbucketCloud) CreatePullRequest(ctx context.Context, repo scm.Repos
 
 	splitRepoFullName := strings.Split(prRepo.FullName(), "/")
 
+	repoOptions := &bitbucket.RepositoryOptions{
+		Owner:    bbc.workspaces[0],
+		RepoSlug: splitRepoFullName[1],
+	}
+	defaultReviewers, _ := bbc.bbClient.Repositories.Repository.ListDefaultReviewers(repoOptions)
+	for _, reviewer := range defaultReviewers.DefaultReviewers {
+		newPR.Reviewers = append(newPR.Reviewers, reviewer.Uuid)
```

Comment:
Have you tried testing with no default reviewers on repo?

Reply:
Yup - I tried just tested it and confirmed it works. Tried create PR with no default reviewers, PR was created successfully with no reviewers on it